modules/ROOT/pages/assembly_tasks.adoc

```diff
@@ -0,0 +1,14 @@
+[]
```

Comment:
Why are these empty square brackets here? Is it for the topic anchor? If so, please add the anchor.